### PR TITLE
election page label changes

### PR DESCRIPTION
**Pull request by umayangag**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -95,6 +95,8 @@ export const TALLY_SHEET_CODE_PRE_34_I_RO = "PRE-34-I-RO";
 export const TALLY_SHEET_CODE_PRE_34_II_RO = "PRE-34-II-RO";
 export const TALLY_SHEET_CODE_PRE_34_PD = "PRE-34-PD";
 export const TALLY_SHEET_CODE_PRE_34_ED = "PRE-34-ED";
+export const TALLY_SHEET_CODE_REVISED_30_PD = "REVISED-30-PD";
+export const TALLY_SHEET_CODE_REVISED_30_ED = "REVISED-30-ED";
 export const TALLY_SHEET_CODE_PRE_34_AI = "PRE-34-AI";
```
Comment (dinukadesilva) [inline src/App.js]: Don't add these here and keep it just in the getTalltSheetCodeStr for now.

File: src/utils/tallySheet.js [hunk 2]
```diff
@@ -34,6 +38,13 @@ export const getElectoralDistrictName = (tallySheet) => {
 
 export function getTallySheetCodeStr({tallySheetCode, election}) {
     let tallySheetCodeStr = tallySheetCode;
+
+    if (tallySheetCode === TALLY_SHEET_CODE_PRE_34_PD) {
+        tallySheetCodeStr = TALLY_SHEET_CODE_REVISED_30_PD
```
Comment (dinukadesilva) [inline src/utils/tallySheet.js]: This has a postal version too :)

File: src/pages/election.js [hunk 3]
```diff
@@ -219,9 +210,9 @@ export default function Election(props) {
                             {election.subElections.map((subElection) => {
                                 const subElectionId = subElection.electionId;
                                 let tallySheetCode = TALLY_SHEET_CODE_PRE_34_PD;
-                                let tallySheetCodeLabel = "PRE 34 PD";
+                                let tallySheetCodeLabel = "Revised 30 PD";
                                 if (subElection.voteType === "Postal") {
-                                    tallySheetCodeLabel = "PRE 34 PV";
```
Comment (dinukadesilva) [inline src/pages/election.js]: Shall we use `getTallySheetCodeStr` method here and remove these.

Comment (umayangag) [reply src/pages/election.js]: done

**Pull request merged**